packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-virtualadmin/nuxeo.defaults

```diff
@@ -1 +1,3 @@
 default.target=.
+
+org.nuxeo.apidoc.apidocAdmin.password=
```

Comment:
how should we set this one?
should it not pas a parameter of the docker image ?

Reply:
This is the nuxeo.defaults conf: i added the variable declaration (used in associated xml in the template) to make the variable existence explicit.

I can setup a default value here if (as per @efge's question) we do not want this to be empty if the variable is not set.

In the end, it will be controlled by the env variable passed to the docker run command, like other variable mappings that are exposed in the nuxeo.prod configuration file in this PR.

Reply:
Provided a default value for it